redfish-core/lib/pcie.hpp

```diff
@@ -79,9 +92,7 @@ inline void requestRoutesSystemPCIeDeviceCollection(App& app)
         .privileges(redfish::privileges::getPCIeDeviceCollection)
         .methods(boost::beast::http::verb::get)(
             [](const crow::Request&,
-               const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
-
-            {
+               const std::shared_ptr<bmcweb::AsyncResp>& asyncResp) {
```

Comment:
ok - so that means a clang-format error slipped in a previous patch.  what I do in that situation is run clang-format before I make any changes, put that in a commit of its own, and then make the changes.  Not doing this just irritates your reviewers (maybe some reviewers don't mind but a lot of them do :-))